@inheritDoc warnings

[WARNING] /home/elharo/google-auth-library-java/appengine/java/com/google/auth/appengine/AppEngineCredentials.java:79: warning - @inheritDoc used but getApplicationDefault() does not override or implement any method.
[WARNING] /home/elharo/google-auth-library-java/appengine/java/com/google/auth/appengine/AppEngineCredentials.java:91: warning - @inheritDoc used but getApplicationDefault(HttpTransportFactory) does not override or implement any method.

--- a/appengine/java/com/google/auth/appengine/AppEngineCredentials.java
+++ b/appengine/java/com/google/auth/appengine/AppEngineCredentials.java
@@ -70,8 +70,6 @@ public class AppEngineCredentials extends GoogleCredentials implements ServiceAc
   private transient AppIdentityService appIdentityService;
 
   /**
-   * {@inheritDoc}
-   *
    * @deprecated AppEngineCredentials should be instantiated via its Builder. See
    *     https://github.com/googleapis/google-auth-library-java#google-auth-library-appengine
    */
@@ -82,8 +80,6 @@ public class AppEngineCredentials extends GoogleCredentials implements ServiceAc
   }
 
   /**
-   * {@inheritDoc}
-   *
    * @deprecated AppEngineCredentials should be instantiated via its Builder. See
    *     https://github.com/googleapis/google-auth-library-java#google-auth-library-appengine
    */

